test/swarm-muxing.node.js

```diff
@@ -9,7 +9,7 @@ chai.use(dirtyChai)
 const parallel = require('async/parallel')
 const TCP = require('libp2p-tcp')
 const WebSockets = require('libp2p-websockets')
-const spdy = require('libp2p-spdy')
+const spdy = require('pull-mplex')
```

Comment:
why changing `spdy` to `pull-mplex` here?

Reply:
Good catch! I was testing locally and forgot to fix that. Now the tests will run against both mplex and spdy